backend/inspirehep/search/mappings/v7/records/experiments.json

```diff
@@ -19,6 +19,30 @@
       "_full_ingestion": {
         "type": "boolean"
       },
+      "accelerator_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "collaboration_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "experiment_short_name_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "experiment_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "institution_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "long_name_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "name_variants_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
+      "legacy_name_search_as_you_type": {
+        "type": "search_as_you_type"
+      },
```

Comment:
alternative: define a `search_as_you_type` object, with all the search fields as subfields, so this becomes easier to exclude in the serializers.